README.md

```diff
@@ -15,7 +15,13 @@ A Force Field Analysis Tools for the Product Field Canvas.
 - `npm run devserver`
 - visit `http://localhost:8080/`
 
-#### ComponentLibrary
+### Run Illustrator Server
+
+The illustrator server renders the product field as svg image. I will be used to create explaing images fpr the product field (e-)book.
```

Comment:
typo "[...] explaing images fpr [...]" should be "[...] explaining images for [...]"